docs/resources/images_image_copy.md

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Image Management Service (IMS)"
+---
+
+# huaweicloud_images_image_copy
+
+Manages an IMS image copy resources within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "source_image_id" {}
+variable "name" {}
+variable "target_region" {}
+variable "agency_name" {}
+
+resource "huaweicloud_images_image_copy" "test" {
+  source_image_id = var.source_image_id
+  name            = var.name
+  target_region   = var.target_region
+  agency_name     = var.agency_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `source_image_id` - (Required, String, ForceNew) Specifies the ID of the copied image.
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of the copy image. The name can contain `1` to `128` characters,
+  only Chinese and English letters, digits, underscore (_), hyphens (-), dots (.) and space are
+  allowed, but it cannot start or end with a space.
+
+* `target_region` - (Required, String, ForceNew) Specifies the target region name.
+  If specified, it means cross-region replication.
+
+* `description` - (Optional, String, ForceNew) Specifies the description of the copy image.
+  Changing this parameter will create a new resource.
+
+* `kms_key_id` - (Optional, String, ForceNew) Specifies the master key used for encrypting an image.
+  Only copying scene within a region is supported.
+  Changing this parameter will create a new resource.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project id of the image.
```

Comment:
this parameter is not supportted in cross-region scene

Reply:
done